### PR TITLE
Updated the nav bar so it looks as expected

### PR DESCRIPTION
**Pull request by florisdobber**
## Why

[Monday.com Ticket](https://code4community-team.monday.com/boards/XXXXXXXX/pulses/XXXXXXX)

What benefit does this bring to the end user? Or, what benefit does this bring to developers working in the codebase?

We needed a nav bar that looks like the designs

## This PR

Describe the changes required and any implementation choices you made to give context to reviewers.

Put the file in a separate folder, changed some file names. Mostly just created the top bar as a Page Header Ant-D component.

## Screenshots

Provide screenshots of any new components, styling changes, or pages.
image
image

## Verification Steps

What steps did you take to verify your changes work? These should be clear enough for someone to be able to clone the branch and follow the steps themselves.

No real logic going on, doesn't work great for mobile yet but much of the website isn't optimized for mobile yet.

### CODE REVIEW

File: src/components/nav-bar/NavBar.tsx [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+const { Paragraph } = Typography;
+
+const NavBar: React.FC = () => {
+  const history = useHistory();
+  const location = useLocation();
+  const path = location.pathname;
+
+  const isLoggedIn = () => {
+    return !(path === '/' || path === '/login' || path === 'signup');
+  };
```
Comment (jackblanc) [inline src/components/nav-bar/NavBar.tsx:15]: This is a bit hacky, let's leave a comment to update it to use a redux selector

Comment (jackblanc) [reply src/components/nav-bar/NavBar.tsx:15]: Actually, you could write such a selector. Ping me if you have questions on how that works, I'm writing docs now.

File: src/components/nav-bar/NavBar.tsx [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+  const BackIcon = () => {
+    const Logo: string = require('../../nav-bar-icon.png');
+    return <img className="back-icon" src={Logo} alt="icon" />;
+  };
```
Comment (jackblanc) [inline src/components/nav-bar/NavBar.tsx:20]: This seems a bit excessive to get a back icon. What's the icon and how come we don't just use one from AntD or an SVG?

File: src/components/nav-bar/NavBar.tsx [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+  return (
+    <PageHeader
+      className="page-header"
+      title="Speak for the Trees"
+      backIcon={<BackIcon />}
+      onBack={() => history.push('/')}
+      extra={isLoggedIn() ? <LoggedInExtra /> : <LandingExtra />}
+    ></PageHeader>
```
Comment (jackblanc) [inline src/components/nav-bar/NavBar.tsx:92]:
```suggestion
    />
```

File: src/components/nav-bar/NavBar.tsx [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+
+  const LoggedInExtra = () => {
+    return (
+      <div className="logged-in-extra">
+        <Paragraph>Jack Blanc</Paragraph>
+        <Dropdown overlay={menu} placement="bottomLeft">
+          <Avatar
+            size="large"
+            icon={<UserOutlined />}
+            style={{ backgroundColor: '#3A681A' }}
```
Comment (jackblanc) [inline src/components/nav-bar/NavBar.tsx:78]: ⛏️ Constants for colors

File: src/components/nav-bar/NavBar.tsx [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+          Log In
+        </Button>
+      </div>
+    );
+  };
+
+  const LoggedInExtra = () => {
+    return (
+      <div className="logged-in-extra">
+        <Paragraph>Jack Blanc</Paragraph>
```
Comment (jackblanc) [inline src/components/nav-bar/NavBar.tsx:73]: Again, leave a comment for anything hardcoded that'll change

File: src/components/nav-bar/nav-bar.less [hunk 6]
```diff
@@ -0,0 +1,26 @@
+.page-header {
```
Comment (jackblanc) [inline src/components/nav-bar/nav-bar.less:1]: We shouldn't be adding any less files at this point. This all needs to be done with styled-components. That should hopefully resolve the need to override any `ant` classes directly, we can't do that.

**Review by jackblanc — COMMENTED** [review]
Let's pair on connecting this to redux

File: src/components/nav-bar/NavBar.tsx (hunk 2 above)
Comment (florisdobber) [reply src/components/nav-bar/NavBar.tsx:20]: I can't get this to work without doing this, and it needs to be this cause it's a SFTT logo really not a back icon

File: src/components/nav-bar/nav-bar.less (hunk 6 above)
Comment (florisdobber) [reply src/components/nav-bar/nav-bar.less:1]: Sorry so no less files at all?

Comment (jackblanc) [reply src/components/nav-bar/nav-bar.less:1]: Correct, it should all be done with styled-components

**Review by jackblanc — APPROVED** [review]

**Pull request merged**